README.md

```diff
@@ -52,6 +52,16 @@ In case you use an exception reporting service (e.g. Bugsnag) you might want to
 define an error handler. See [Customizing the error
 handler](#customizing-the-error-handler) for more information.
 
+**ActiveJob Dependency**
+
+The Maintenance Tasks framework relies on ActiveJob behind the scenes to run
+Tasks. The default queuing backend for ActiveJob is
+[asynchronous](https://api.rubyonrails.org/classes/ActiveJob/QueueAdapters/AsyncAdapter.html).
```

Comment:
Can we pull out URLs from the text and use markdown references like I did in other places in the README?

Reply:
Yes, although it still goes over the 80 character limit 😢